server/methods/core/orders.js

```diff
@@ -324,6 +324,83 @@ Meteor.methods({
    */
   "orders/sendNotification": function (order) {
     check(order, Object);
+    const shoppersPhone = order.billing[0].address.phone;
+
+    //  Loop through orders to get vendor information and send notification to them
+    const orderedItems = order.items;
+    let orderedProducts = "";
+    let vendorPhones = [];
+    for (let i = 0; i < orderedItems.length; i += 1) {
+      orderedProducts += ` ${orderedItems[i].title},`;
+      const productVendor = Accounts.find({ _id: orderedItems[i].vendorId }).fetch();
+      Logger.info("Vendor Object", productVendor);
+      vendorPhones.push(productVendor[0].profile.vendorDetails.vendorPhone);
+    }
+    Logger.info("Vendor Phones: ", vendorPhones);
+
+
+    Logger.info("ORDER", order.items);
+    const smsContent = {
+      to: shoppersPhone,
+      message: `Receipt for ${orderedProducts} has been received. Thanks.`
+    };
+
+    if (order.workflow.status === "new") {
+      Meteor.call("send/smsAlert", smsContent, (error, result) => {
+        if (error) {
+          Logger.warn("ERROR", error);
+        } else {
+          Logger.info("SMS SENT");
+        }
+      });
+
+      vendorAlertMessage = "You have pending orders on your Banasko-RC store";
+
+      // Filter out the duplicate values
+      vendorPhones = vendorPhones.filter((item, index, inputArray) => {
+        return inputArray.indexOf(item) == index;
+      });
+
+      vendorPhones.forEach((number) => {
+        const vendorSmsContent = {};
+        vendorSmsContent.to = number;
+        vendorSmsContent.message = vendorAlertMessage;
+        Meteor.call("send/smsAlert", vendorSmsContent, (error) => {
+          if (error) {
+            Logger.warn("ERROR", error);
+          } else {
+            Logger.info("SMS SENT");
+          }
+        });
+      });
+    } else if (order.workflow.status === "coreOrderWorkflow/processing") {
+      smsContent.message = "Your orders is on the way and will soon be delivered.";
+      Meteor.call("send/smsAlert", smsContent, (error) => {
+        if (error) {
+          Logger.warn("ERROR", error);
+        } else {
+          Logger.info("SMS SENT");
+        }
+      });
+    } else if (order.workflow.status === "coreOrderWorkflow/completed") {
+      smsContent.message = "Your orders on Banasko-RC has been shipped, thanks";
+      Meteor.call("send/smsAlert", smsContent, (error) => {
```

Comment:
since this method is repeated in other parts of the code, could you abstract it and reuse? keep it DRY